packages/grafana-data/rollup.config.ts

```diff
@@ -0,0 +1,50 @@
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import sourceMaps from 'rollup-plugin-sourcemaps';
+import { terser } from 'rollup-plugin-terser';
+
+const pkg = require('./package.json');
+
+const libraryName = pkg.name;
+
+const buildCjsPackage = ({ env }) => {
```

Comment:
I think this config, as it's shared between grafana/* packages, could be a part of grafana/toolkit. WDYT @ryantxu?

Reply:
agree -- but in a different PR